timestream.lua

```diff
@@ -0,0 +1,393 @@
+-- speeds up the calendar, units, or both
+
+--[====[
+
+timestream
+==========
+This only affects fortress mode.
+
+Usage:
+:timestream <scalar> <fps> <simulate units y/n>
+Examples:
```

Comment:
The formatting here is off - you need more blank lines for it to format correctly. This is what it currently looks like when built (see [here](https://docs.dfhack.org/en/latest/docs/Documentation.html) for information on building the docs):

![image](https://user-images.githubusercontent.com/3719547/93963301-ca238280-fd2a-11ea-85fc-38bcda0ec322.png)

I made a couple changes locally that made it look better (note that the `:` at the beginning of a line is only for "definition lists", which we use for some scripts' arguments, not for code blocks - those require `::` on the previous line, and need to be separated by a blank line):

```suggestion
Usage::

    timestream <scalar> <fps> <simulate units y/n>

Examples:

```


Here's what it looks like after my changes: 
![image](https://user-images.githubusercontent.com/3719547/93963210-921c3f80-fd2a-11ea-9cc3-8d05a9bf74d4.png)

Reply:
Done, thanks.